python/intro/README.md

```diff
@@ -0,0 +1,405 @@
+# Introduction to Fn with Python
+Fn is a lightweight Docker-based serverless functions platform you can run on
+your laptop, server, or cloud.  In this introductory tutorial we'll walk through
+developing a function using the Python programming language (without installing
+any Python tools!) and deploying that function to a local Fn server.  We'll also
+learn about the core Fn concepts like applications and routes.
+
+### Before you Begin
+* Set aside about 15 minutes to complete this tutorial.
+* Make sure Fn server is up and running by completing the [Install and Start Fn Tutorial](../../install/README.md).
+
+> As you make your way through this tutorial, look out for this icon.
+![](images/userinput.png) Whenever you see it, it's time for you to
+perform an action.
+
+## Your First Function
+
+Let's start with a very simple "hello world" function written in
+[Python](https://www.python.org/). Don't worry, you don't need to know Python!
+In fact you don't even need to have Python installed on your development machine
+as Fn provides the necessary Python compiler and tools as a Docker container.
```

Comment:
python is not a compiled language, it is an interpreted one, so fix this:
`Python compiler` to `Python interpreter`

Reply:
`and tools` to `and build-tools`

Reply:
Fixed.